TESTS/usb_device/basic/USBTester.cpp

```diff
@@ -45,6 +43,7 @@
 
 #define EVENT_READY (1 << 0)
 
+using namespace std::chrono_literals;
```

Comment:
Don't forget you can `using namespace std::chrono` instead to save typing `std::chrono::duration`. or `std::chrono::milliseconds`.